### ISSUE TITLE
Normalize data before Parametric UMAP

### ISSUE THREAD

**Issue by JoshuaHess12** [JoshuaHess12/hdi-prep#13]
warnings raise to normalize data between 0-1 before computing cross-entropy loss in neural network
